chain-impl-mockchain/src/config.rs

```diff
@@ -328,25 +334,39 @@ impl ConfigParamVariant for RewardParams {
                 ratio,
                 epoch_start,
                 epoch_rate,
+                rewards_limit,
+                npools,
+                npools_threshold,
             } => ByteBuilder::new()
                 .u8(1)
                 .u64(*constant)
                 .u64(ratio.numerator)
                 .u64(ratio.denominator.get())
                 .u32(*epoch_start)
-                .u32(epoch_rate.get()),
+                .u32(epoch_rate.get())
+                .u64(rewards_limit.numerator)
+                .u64(rewards_limit.denominator.get())
+                .u32(npools.get())
+                .u32(npools_threshold.get()),
             RewardParams::Halving {
                 constant,
                 ratio,
                 epoch_start,
                 epoch_rate,
+                rewards_limit,
+                npools,
+                npools_threshold,
```

Comment:
I will fix this.  I'm not seeing it in my copy of the code, so it may be the result of a merge error.  It would have passed the tests, of course, since we're using the same values.  I will change that, so we can spot any further error.

Reply:
Jared fixed this one already